.github/workflows/action.yml

```diff
@@ -1,14 +0,0 @@
-on: push
```

Comment:
this snippet is already in the readme- let's remove it from the directory structure so it doesn't actually run as an action

Reply:
hmm yeah why did we have this here. wonder if i was just using it to test things

Reply:
I added this when trying to get the action working for the launch, and there was a lot of confusion at the time of what file we needed to add and how to specify it....